### ISSUE TITLE
RFID: furi_check failed crash on read raw rfid data attempt.

### ISSUE THREAD

**Issue by mxcdoam** [flipperdevices/flipperzero-firmware#4046]
### Describe the bug.

Flipper is crashing with "furi_check failed" while attempting a RFID RAW read. I found this issue while trying to get a raw record from not supported lf tag, but the crash also happens if no tag is present at all. RAW RFID reading of supported tag (EM4100) does not show any issues. 
Seems like there is no useful info in the log with debug/trace levels.
FW: 1.1.2 (release)

### Reproduction

1. Go to Settings -> System -> Debug and toggle it to "ON"
2. Go to RFID app -> Extra Actions -> Read RAW RFID data
3. Put an unsupported lf tag to the back of the flipper. Enter a filename, press "save" and then press "OK". 
4. Repeat all the above except on step 3 do not put any tag to the back of the flipper.

**Comment by skotopes** [comment]
@mxcdoam Can't reproduce your issue. Do you have debugger? Can you show crash back trace?

**Comment by mxcdoam** [comment]
No, sorry. But I flashed the latest dev build and got the error screen instead of a crash. It was saying that the error is probably due to my SD card failing. I changed the card and the error is gone, so I'm closing this issue.
